documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -136,11 +140,11 @@ $ cat << EOF | oc apply -f -
 apiVersion: virt.konveyor.io/v1alpha1
 kind: Migration
 metadata:
-  name: migration-001
+  name: migration_name
   namespace: {namespace}
 spec:
   plan:
-    name: test-migration-plan
+    name: migration_plan
```

Comment:
I think it can be good to explain here that this name is the name of the migration plan that we'd like to run/start,

Reply:
Good idea.